vendor-extra/Csa/GuzzleBundle/GuzzleHttp/Middleware/MockMiddleware.php

```diff
@@ -45,7 +45,11 @@ public function __invoke(callable $handler)
 
             try {
                 if (null === $response = $this->adapter->fetch($request)) {
-                    throw new \RuntimeException('Record not found.');
```

Comment:
Could you match the change in https://github.com/csarrazi/CsaGuzzleBundle/commit/d250011dbfbfb558d579e2cc7d340ae9bd532106 exactly?